Makefile

```diff
@@ -0,0 +1,34 @@
+PREFIX ?= /usr/local
+BASH_COMPLETION_DIR := $(shell pkg-config --variable=completionsdir bash-completion) 
+# pkg-config adds a space for some reason, we have to strip it off.
+BASH_COMPLETION_DIR := $(strip $(BASH_COMPLETION_DIR))
+USERDIR ?= $(HOME)
+
+# The @ symbols make the output silent.
+# the \033[0m stuff is ansi color escape codes.
+
+install: 
+	@if [ -d $(BASH_COMPLETION_DIR) ]; then \
+		cp notes.bash_completion "$(BASH_COMPLETION_DIR)/notes"; \
+	else \
+		echo "Bash Completion was not installed, because the directory was" \
+		"not found. if you have bash completion installed, follow the" \
+		"README (https://github.com/pimterry/notes#installing-bash-completion)" \
+		"to manually install it."; \
+	fi # Small test for bash completion support
+	@install -m755 -D notes $(PREFIX)/bin/notes
+	@install -m777 -D config $(USERDIR)/.config/notes/config
+	@install -D notes.1 $(PREFIX)/share/man/man1/notes.1
+	@mandb 1>/dev/nulli 2>&1 # Fail silently if we don't have a mandb
+	@echo -e "Notes has been installed to \033[0;32m$(PREFIX)/bin/notes." \
```

Comment:
`echo -e` isn't supported in all shells, and I think generally [isn't recommended](http://askubuntu.com/a/537987/116547). We should probably move all of this to printf really, but I'd be fine with a working simple `echo` solution. In zsh on OSX this prefixes this message with `-e ` for me right now, which isn't great.